domain/modelconfig/service/service.go

```diff
@@ -31,10 +31,10 @@ type State interface {
 	ProviderState
 	SpaceValidatorState
 
-	// AgentVersion returns the current models agent version. If no agent
-	// version has been set for the current model then a error satisfying
-	// [errors.NotFound] is returned.
-	AgentVersion(context.Context) (string, error)
+	// GetModelAgentVersionAndStream returns the current models set agent
+	// version and stream. If no agent version or stream has ben set then an
```

Comment:
been

nit: can we conform to the doc pattern we are trying to standardise on
```
The following errors can be expected:
...
```

Reply:
Done & Done